.github/workflows/githubRelease.yml

```diff
@@ -8,6 +8,10 @@ on:
         default: false
         description: use a prerelease instead of a regular release
 
+# -----------------------------------------------------------------------
+# NOTE: This workflow is deprecated in favor of create-github-release.yml
+# -----------------------------------------------------------------------
```

Comment:
`create-github-release.yml` doesn't exist in this repo, do we want to copy it or should users just use it from `salesforcecli`?

Reply:
Good catch, copy/paste. I'll update